src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -120,6 +120,7 @@ func (client *MobileClient) updateConfig() error {
 	var buf []byte
 	var err error
 	if buf, err = pullConfigFile(client.fronter); err != nil {
+		log.Fatalf("Could not update config: '%v'", err)
```

Comment:
Good catch. No not intended. I think we should actually be using ```getlantern/golog``` here and Errorf. We should do a new build @atavism.

Any reason we're not using getlantern/golog in lantern-android?